department-of-veterans-affairs/va.gov-cms
# VACMS-7309: Disallow HTML characters from character count on text fields

## Description

Closes #7309 .

## Testing done
Edited a vet center featured content section to add a blurb with a link and then removed the link to ensure the counter was not counting html characters. 

## Screenshots
Example with url: 
![image](https://user-images.githubusercontent.com/8375335/147968743-32164343-a347-47f0-8437-0a28ac5a9b78.png)

Same example with no url: 
![image](https://user-images.githubusercontent.com/8375335/147968761-ea7f6a08-d273-458a-979b-ebec157452df.png)


## QA steps

As a cms editor I can
1. Edit a Vet Center
2. Add "Featured Content"
4. Type more than 240 characters to within the description including a URL
5. See error message of "Description cannot be longer than 240 characters but is currently (number that's greater than 240) characters long
6. Remove characters until you're under 240 characters while keeping the URL intact
7. Remove the URL and confirm the character count does not change. 

1. Then validate Acceptance Criteria from issue
- [x] All text counters applied to rich text fields have been configured to ignore html characters

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
